identity_iota_core/src/rebased/client/read_only.rs

```diff
@@ -7,14 +7,14 @@ use std::ops::Deref;
 use std::pin::Pin;
 use std::str::FromStr;
 
-use crate::rebased::iota;
+use crate::rebased::{iota, rebased_err};
```

Comment:
Maybe something like this?

```suggestion
use crate::rebased::iota;
#[cfg(not(target_arch = "wasm32"))]
use crate::rebased::rebased_err;
```